crates/circuits/poseidon2-air/src/lib.rs

```diff
@@ -112,3 +113,262 @@ pub type Plonky3Poseidon2Executor<F, LinearLayers> = Poseidon2<
     POSEIDON2_WIDTH,
     BABY_BEAR_POSEIDON2_SBOX_DEGREE,
 >;
+
```

Comment:
Both this code block, as well as the one added in `Keccak-Air` come from `Plonky3` and are the result of importing `BaseAir`